features/step_definitions/run_steps.rb

```diff
@@ -63,8 +63,9 @@
 end
 
 
-Then(/^I see "(.+?)"$/) do |output|
-  expect(@last_run_result.out).to eql "#{output}\n"
+Then(/^I(?: do|( don't))? see "(.*)"$/) do |verb, string|
+  method = verb == "don't" ? :not_to : :to
+  expect(@last_run_result.out).public_send(method, include(string))
 end
```

Comment:
I updated the step to check for exact output instead of just including.
If you want a step to check that the output includes something (which I'm against, I think we should check the exact output always), please add a new step.